lawnchair/src/app/lawnchair/ui/preferences/components/colorpreference/ColorSelectionPreference.kt

```diff
@@ -44,14 +44,19 @@ fun NavGraphBuilder.colorSelectionGraph(route: String) {
             val preferenceManager2 = preferenceManager2()
             val pref = when (prefKey) {
                 preferenceManager2.accentColor.key.name -> preferenceManager2.accentColor
+                preferenceManager2.notificationDotColor.key.name -> preferenceManager2.notificationDotColor
                 else -> return@composable
             }
             val label = when (prefKey) {
                 preferenceManager2.accentColor.key.name -> stringResource(id = R.string.accent_color)
+                preferenceManager2.notificationDotColor.key.name -> stringResource(id = R.string.notification_dots_color)
                 else -> return@composable
             }
-            ColorSelection(label = label, preference = pref)
-
+            ColorSelection(
+                label = label,
+                preference = pref,
+                dynamicEntries = if (prefKey == preferenceManager2.notificationDotColor.key.name) dynamicColorsForNotificationDot else dynamicColors,
```

Comment:
Maybe extract this to a `when` statement? Like ``pref` and `label`?

Reply:
Makes sense. Not sure why I didn't do.